lib/que/sql.rb

```diff
@@ -162,6 +162,30 @@ module Que
         WHERE locktype = 'advisory'
       ) pg USING (job_id)
     },
+
+    unlock_job: %{
+      SELECT pg_advisory_unlock($1)
+    },
+
+    find_job_to_lock: %{
+      SELECT 
+        queue, 
+        priority, 
+        run_at, 
+        job_id, 
+        job_class, 
+        retryable, 
+        args, 
+        error_count,
+        extract(epoch from (now() - run_at)) as latency
+      FROM que_jobs
+      WHERE queue = $1::text
+            AND run_at <= now()
+            AND retryable = true
+            AND job_id >= $2
+            ORDER BY priority, run_at, job_id
+            LIMIT 1
```

Comment:
I'm assuming that the primary key is sufficient for this query to be efficient, is that correct?

Reply:
yeah, that's my assumption.

Reply:
Sounds good. We can run a query plan once this is deployed